cmd/tink-cli/cmd/hardware/ip.go

```diff
@@ -33,15 +34,20 @@ var ipCmd = &cobra.Command{
 			if err != nil {
 				log.Fatal(err)
 			}
-			b, err := json.Marshal(hw)
-			if err != nil {
-				log.Fatal(err)
+			if hw.GetId() == "" {
+				log.Fatal("IP address not found in the database ", ip)
 			}
-			fmt.Println(string(b))
+			printOutput(data, hw, ip)
 		}
 	},
 }
 
+func addIPFlags() {
+	flags := ipCmd.Flags()
+	flags.BoolVarP(&data, "details", "d", false, "Provide the complete hardware details in json format")
```

Comment:
The flag description should start with a small case. That way it will be consistent across the CLI.

Reply:
ok